_posts/2019-10-11-nottingham-tech-community.md

```diff
@@ -0,0 +1,43 @@
+---
+layout: post
+title: The Nottingham Tech Community
+description: "A look at the depth of meetups"
+date: 2019-10-10
+cover: "/assets/header_image.jpg"
+author: Jonathan Relf
+categories: [community, hacktoberfest]
+comments: false
+share: true
+---
+
+Nottingham is a city with hidden depths. Built around sandstone outcrops there are many tunnels and shelters hewn into the bedrock that are found all around the area. Some who grew up with the legend of *Robin Hood*, whose has close associations with Nottingham, believe the Robin & his followers hid in caves occasionally.
+
+Fast forward to more recent times and the [TechNation 2018 report insights for Nottingham](https://technation.io/insights/report-2018/nottingham/_) describes the city as a ***"Meetup Hub"*** with a £1.3 billion tech turnover.
+
+Nottingham's not-so-secret weapon is the strength of talent it has across a wide variety of tech sectors. From gaming to science; pharmacuticals to finance; startups to world leaders, there is a lot going on in and around the city. With so much going on it can sometimes feel like work happens in isolated 'caves'. It is Nottingham's vibrant *'meetup hub'* that is really unlocking extra value as we come together, learn more about each other, share our experiences, and meet new people.
+
+![Statue of Robin Hood - Image by Ian Wilson from Pixabay](/assets/img/2019-10-11/robin-hood.jpg)
```

Comment:
Would it be ok to put this in HTML so that the alt text works the same as with the other blog posts please?

```
<img src="/assets/img/2019-10-11/robin-hood.jpg" alt="Statue of Robin Hood - Image by Ian Wilson from Pixabay" width="100%"/>
<center>
<i>Statue of Robin Hood - Image by Ian Wilson from Pixabay</i>
</center>
<br/>
```